package.json

```diff
@@ -12,7 +12,7 @@
   "author": "Vladimir Agafonkin",
   "repository": {
     "type": "git",
-    "url": "git://github.com/mapbox/geojson-vt.git"
+    "url": "git://github.com/montzkie18/geojson-vt.git"
```

Comment:
This is included in the PR accidentally, right?

Reply:
lol yes. tried to publish it to npm temporarily to test in another project. reverted it here